constants/shared.go

```diff
@@ -3,8 +3,20 @@ package constants
 import "time"
 
 const (
-	// EpochLength is the number of blocks in an epoch for MadNet
-	EpochLength uint32 = 1024
+	// EpochLength is the number of blocks in an epoch for MadNet. A lower calue can be used during development, to quickly test behaviour at the epoch boundary.
+	EpochLength uint32 = 32 // 10
+
+	// MinEthSnapshotInterval is the minimum number of Ethereum blocks to be between each snapshot
+	MinEthSnapshotInterval int = int(EpochLength) / 8
+
+	// SnapshotDesperationDelay is after how many Etereum blocks more validators will start being allowed. Highly dependent on EpochLength
+	SnapshotDesperationDelay int = 18 // 600
```

Comment:
The snapshots parameters also need to be added to the `./scripts/base-files/DeploymentArgsTemplate` file. Sorry for not mentioning this before, this is the file where we set the constructors/initialize parameters for the deployed smart contracts.